modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/ActiveActorTest.java

```diff
@@ -193,453 +144,22 @@ private boolean checkSingleActiveActor(String leaderName) {
-        int nodes = 3;
-        Predicate<NetworkAddress> isServerAddress = addr -> true;
-
-        TopologyAwareRaftGroupService raftClient = startCluster(
-                testInfo,
-                clusterServices,
-                raftServers,
-                isServerAddress,
-                nodes,
-                PORT_BASE
-        );
-
-        raftClient.refreshLeader().get();
-
-        var clientClusterService = clusterService(testInfo, PORT_BASE + nodes + 1, new StaticNodeFinder(getNetworkAddresses(nodes)));
-        clientClusterService.start();
-
-        TopologyAwareRaftGroupService raftClientNoInitialNotify = startTopologyAwareClient(
-                clientClusterService,
-                clusterServices,
-                isServerAddress,
-                nodes,
-                null,
-                false
-        );
-
-        AtomicReference<ClusterNode> leaderRef = new AtomicReference<>();
-        AtomicReference<ClusterNode> leaderRefNoInitialNotify = new AtomicReference<>();
-        AtomicInteger callsCount = new AtomicInteger();
-
-        raftClient.subscribeLeader((node, term) -> leaderRef.set(node));
-
-        for (int i = 0; i < 2; i++) {
-            raftClientNoInitialNotify.unsubscribeLeader();
-
-            raftClientNoInitialNotify.subscribeLeader((node, term) -> {
-                callsCount.incrementAndGet();
-                leaderRefNoInitialNotify.set(node);
-            });
-        }
-
-        assertTrue(callsCount.get() <= 1);
-
-        assertTrue(waitForCondition(() -> leaderRef.get() != null, 10_000));
-
-        ClusterNode leader = leaderRef.get();
-
-        assertNotNull(leader);
-
-        log.info("Leader: " + leader);
-
-        afterInitCheckConditionWithWait(leader.name());
-
-        Peer newLeaderPeer = raftClient.peers().stream().filter(peer -> !leader.name().equals(peer.consistentId())).findAny().get();
-
-        log.info("Peer to transfer leader: " + newLeaderPeer);
-
-        raftClient.transferLeadership(newLeaderPeer).get();
-
-        String leaderId = newLeaderPeer.consistentId();
-
-        assertTrue(waitForCondition(() -> leaderId.equals(leaderRef.get().name()), 10_000));
-        assertTrue(waitForCondition(
-                () -> leaderRefNoInitialNotify.get() != null && leaderId.equals(leaderRefNoInitialNotify.get().name()), 1000)
-        );
-
-        log.info("New Leader: " + leaderRef.get());
-
-        afterLeaderChangeCheckConditionWithWait(leaderRef.get().name());
-
-        raftClient.refreshLeader().get();
-
-        assertEquals(raftClient.leader().consistentId(), leaderRef.get().name());
-
-        stopCluster(clusterServices, raftServers, List.of(raftClient, raftClientNoInitialNotify), nodes);
-        clientClusterService.stop();
+    @Override
+    protected void afterClusterInit(String leaderName) throws InterruptedException {
+        assertTrue(waitForCondition(() -> checkSingleActiveActor(leaderName), 10_000));
```

Comment:
please extract `10_000` to a common timeout value all over your changes

Reply:
done